processing/src/main/java/org/apache/druid/query/aggregation/last/FloatLastVectorAggregator.java

```diff
@@ -47,9 +47,9 @@ void putValue(ByteBuffer buf, int position, int index)
   }
 
   @Override
-  void initValue(ByteBuffer buf, int position)
+  public void initValue(ByteBuffer buf, int position)
   {
-    buf.putFloat(position, 0);
+    buf.putFloat(position, Float.NEGATIVE_INFINITY);
```

Comment:
hmm shouldn't this be 0? `FloatLastAggregator` initializes to 0

```suggestion
    buf.putFloat(position, 0);
```